### DISCUSSION TITLE
DropwizardApacheConnector to support setting proxy from JerseyClientRequest

### DISCUSSION THREAD

**Discussion by tomaszzasada** [dropwizard/dropwizard#6856] [Q&A]
Setting `ClientProperties.PROXY_URI` in Jersey client **does not** result in setting the proxy in underlaying RequestConfig of http client.

So something like this **is not supported**:
```
client.target(...)
.request(MediaType.APPLICATION_JSON)
.property(ClientProperties.PROXY_URI, "....")
...
```
It would be great if it was...

**Comment by zUniQueX** [comment]
Hi @tomaszzasada. We don't evaluate the Jersey client property for the proxy uri, but you can set a proxy via the `#using(HttpClientConfiguration)` method.

**Reply by joschi** [reply]
@tomaszzasada As @zUniQueX pointed out, `ClientProperties.PROXY_URI` has to be set in the client configuration and cannot be changed per-request.

See also: https://eclipse-ee4j.github.io/jersey.github.io/apidocs/2.39/jersey/org/glassfish/jersey/apache/connector/ApacheConnectorProvider.html
